test/dlc_tests/benchmark/sagemaker/tensorflow/training/test_trcomp_performance.py

```diff
@@ -88,7 +88,7 @@ def framework_version(tensorflow_training):
 
 @pytest.fixture(autouse=True)
 def smtrcomp_only(framework_version, tensorflow_training, request):
-    if Version(framework_version) in SpecifierSet(">=2.9.1"):
+    if Version(framework_version) in SpecifierSet("<2.9.1"):
```

Comment:
can we wrap this logic into a decorator and share this across all tests?

Reply:
It is already shared and applied to all tests in this file.

https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request